dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -185,4 +185,7 @@ setHtmlContentTypeOnError.name=Disable text/html content type on error responses
 setHtmlContentTypeOnError.desc=When this property is false, the webcontainer will not set the response's content type header during the error handling process. An application is responsible to set the response's content type. The default value (true) sets the content type to "text/html".
 
 excludeAllHandledTypesClasses.name=Disable returning handlesTypes parameters
-excludeAllHandledTypesClasses.desc=When this property is set to true, during startup, ServletContainerInitializer implementors that use the HandlesTypes annotation do not receive classes that are specified as HandlesTypes parameters.
\ No newline at end of file
+excludeAllHandledTypesClasses.desc=When this property is set to true, during startup, ServletContainerInitializer implementors that use the HandlesTypes annotation do not receive classes that are specified as HandlesTypes parameters.
+
+closeWrappedResponseOutputAfterForward.name=Close wrapped response output after forward
+closeWrappedResponseOutputAfterForward.desc=When this property is set to true, the wrapped response's output stream will be closed before exiting the dispatch forward.  The default value is false. For servlet-5.0 and higher, the default value is true. 
```

Comment:
When this property is set to true, the wrapped response's output stream will be closed before exiting the dispatch forward
--->
When this property is set to true, the output stream from the wrapped response is closed before it exits the dispatch forward.